Tanasee/Tanasee1.xml

```diff
@@ -53,6 +136,1078 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <locus from="22r" to="238v"/>
+                                    <ab type="pricking">Pricking is partly visible.</ab>
+                                    <ab type="ruling">Ruling is partly visible.</ab>
+                                    <ab type="CruxAnsata">Many pages are decorated with several crux ansata.</ab>
+                                </layout>
+                            </layoutDesc>  
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="ink">Black.</seg>
+                                <seg type="script">Angular script with triangular, rounded loops.</seg>
+                                <date notBefore="1301" notAfter="1500">Script of the fourteenth to fifteenth century.</date>
+                            </handNote>     
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="frame">
+                                <locus target="#6r"/>
+                                <desc>Finely illuminated frame as described and reproduced in <bibl><ptr target="bm:Heldman1972miniatures"/>
+                                    <citedRange>Figure 85</citedRange></bibl> surrounding the text of the <ref target="LIT1349EpistlEusebius"/> 
+                                    as described in <ref target="#ms_i1"/></desc>
+                            </decoNote>
+                            <decoNote xml:id="d2" type="miniature">
+                                <locus target="#6v"/>
+                                <desc>Finely illuminated frame, that was described and reproduced in 
+                                    <bibl><ptr target="bm:LeroyAeth1968"/><citedRange>Abb. 430a</citedRange></bibl> and
+                                    <bibl><ptr target="bm:Heldman1972miniatures"/><citedRange>Figure 86</citedRange></bibl> 
+                                    surrounding the text of the <ref target="LIT1349EpistlEusebius"/> in <ref target="#ms_i1"/>.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d3" type="miniature">
+                                <locus target="#7r"/>
+                                <desc>Finely illuminated frame, that was described and reproduced in 
+                                    <bibl><ptr target="bm:Leroy1962Recherches"/><citedRange>Figure 7</citedRange></bibl>, surrounding 
+                                    the text of <ref target="LIT1349EpistlEusebius"/> as described in <ref target="#ms_i1"/>.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d4" type="frame">
+                                <locus target="#7v"/>
+                                <desc>Finely illuminated frame surrounding <ref target="LIT1224Canons"/> as described in <ref target="#ms_i2"/>.</desc>
```

Comment:
`ref type="work"`

Reply:
Do I need to add or replace it? Is it correct to add it to result in: 
`<ref type="work" target="LIT1224Canons"/>`

Reply:
I think, it happened more often in this record.